ts/src/base/Exchange.ts

```diff
@@ -3775,13 +3776,49 @@ export default class Exchange {
         return this.safeString (market, 'symbol', symbol);
     }
 
+    handleParam (params: object, paramName: string, defaultValue = undefined) {
```

Comment:
this is new method which i've added and already reusing it in other exchanges too. there is specific case when we only need to get something from params and not from exchange.options

Reply:
@ttodua I'm not sure I love it, because it's not typed at all, value can be anything and if in the future we want to enable the noIMplicitAny rule we will have to refactor it

Reply:
what do you think? if we enable `noIMplicitAny ` in future, we would anyway need to make huge refactor in lib, no?

Reply:
we're doing it gradually, with the safeDict/safeList, typing arguments/return types, etc

Reply:
ok, if so, i can add a dedicated method, like `handleCallerMethodName`? wdyt?

Reply:
here is breakdown:
- we have not used that term before, but it is something that we have been missing and is needed now and onwards. ofc, we can change to other name, as you prefer, but the idea is the following:
- look at https://github.com/ttodua/ccxt/blob/gate-watchtickers-symbols/ts/src/pro/gate.ts#L290 
 or in any method in blofin (i.e. https://github.com/ttodua/ccxt/blob/blofin-ws/ts/src/pro/blofin.ts#L78 or other watch methods) which are routed through **watchXyzForSymbols** methods. **watchXyzForSymbols** internally are doing many things and we might be throwing exceptions , like "binance watchTradesForSymbols() - you are doing wrong blabla" , so why showing `watchTradesForSymbols` while user might be using `watchTrades` ?  so we somehow need to make call-chain aware of which method is being called as a first entry watch method
 - not only for exceptions, but also depending the entry watch method we are resolving  `handleOptionsAndParams` (or other `handleXyz` methods) from where we need to get `exchange.options['watchTrades']` and get it's config, and not `exchange.options['watchTradesForSymbols']`. 
 
 so, thats why we need to have it.

Reply:
i've now pushed an edit for the name/type and changed it to `handleParamString` method, which has string type.

Reply:
I think we can keep it for now, but I'm not a big fan of providing ccxt-only values back and forth through `params`.